### ISSUE TITLE
Cannot find path 'C:\Applications\TestFramework\TestLibraries\' because it does not exist.

### ISSUE THREAD

**Issue by borisimple** [microsoft/navcontainerhelper#1257]
BcContainerHelper is @ 1.0.4 version.

```
        New-BCContainer `
            -accept_eula `
            -imageName 'w1' `
            -updateHosts `
            -artifactUrl Get-BCArtifactUrl -type 'OnPrem' -version '' -country 'w1' -select 'Latest' `
            -containerName 'w1' `
            -auth NavUserPassword `
            -credential $cred `
            -licenseFile $tempLicensePath `
            -includeTestToolkit `
            -includeTestLibrariesOnly
```

And the error is:
```
Cannot find path 'C:\Applications\TestFramework\TestLibraries\' because it does not exist at <ScriptBlock>, <No file>: line 4

**Comment by freddydk** [comment]
Full output please.

**Comment by borisimple** [comment]
> Full output please.

```
BcContainerHelper is version 1.0.4
BcContainerHelper is running as administrator
Host is Microsoft  - 1909
Docker Client Version is 19.03.12
Docker Server Version is 19.03.12
Removing container w1
Removing w1 from host hosts file
Removing w1-* from host hosts file
Removing C:\ProgramData\BcContainerHelper\Extensions\w1
ArtifactUrl and ImageName specified
Fetching all docker images
Image w1:onprem-16.4.14693.15445-w1 already exists
Using image w1:onprem-16.4.14693.15445-w1
Creating Container w1
Version: 16.4.14693.15445-w1
Style: onprem
Multitenant: No
Platform: 16.0.14648.15420
Generic Tag: 0.1.0.19
Container OS Version: 10.0.18363.1016 (1909)
Host OS Version: 10.0.18363.1016 (1909)
Using process isolation
Using locale en-US
Disabling the standard eventlog dump to container log every 2 seconds (use -dumpEventLog to enable)
Using license file C:\dev\365 BC\Scripts\eqm_365_license.flf
Files in C:\ProgramData\BcContainerHelper\Extensions\w1\my:
- AdditionalOutput.ps1
- license.flf
- MainLoop.ps1
- SetupVariables.ps1
- updatehosts.ps1
Creating container w1 from image w1:onprem-16.4.14693.15445-w1
e014e88b2953194d978e4a5fb984a2fe6c16cc21c40fa887774c63035a57cdef
Waiting for container w1 to be ready
Initializing...
Setting host.docker.internal to 192.168.1.219 in container hosts file (copy from host hosts file)
Setting gateway.docker.internal to 192.168.1.219 in container hosts file (copy from host hosts file)
Setting kubernetes.docker.internal to 127.0.0.1 in container hosts file (copy from host hosts file)
Setting host.containerhelper.internal to 172.20.80.1 in container hosts file
Starting Container
Hostname is w1
PublicDnsName is w1
Using NavUserPassword Authentication
Starting Local SQL Server
Starting Internet Information Server
Creating Self Signed Certificate
Self Signed Certificate Thumbprint E4D99FC7CA925BCC4FC4E604433490EA695B0C77
Modifying Service Tier Config File with Instance Specific Settings
Starting Service Tier
Registering event sources
Creating DotNetCore Web Server Instance
Using license file 'c:\run\my\license.flf'
Import License
Creating http download site
Setting SA Password and enabling SA
Creating admin as SQL User and add to sysadmin
Creating SUPER user
WARNING: The password that you entered does not meet the minimum requirements. 
It should be at least 8 characters long and contain at least one uppercase 
letter, one lowercase letter, and one number.
Container IP Address: 172.20.91.144
Container Hostname  : w1
Container Dns Name  : w1
Web Client          : http://w1/BC/
Dev. Server         : http://w1
Dev. ServerInstance : BC
Setting w1 to 172.20.91.144 in host hosts file

Files:
http://w1:8080/ALLanguage.vsix

Initialization took 75 seconds
Ready for connections!
Reading CustomSettings.config from w1
Creating Desktop Shortcuts for w1
Cannot find path 'C:\Applications\TestFramework\TestLibraries\' because it does not exist.
at <ScriptBlock>, <No file>: line 4
Cannot find path 'C:\Applications\TestFramework\TestLibraries\' because it does not exist.

**Comment by freddydk** [comment]
That was a bug in generic 0.1.0.19 (race condition with some builds)
I have just published 0.1.0.20 - re-run the script and it should pick up the new generic. rebuild and run.
Sorry

**Comment by borisimple** [comment]
Thanks for the quick feedback. It works fine now 👍🏻
